darwin/DarwinMachine.c

```diff
@@ -125,3 +135,34 @@ bool Machine_isCPUonline(const Machine* host, unsigned int id) {
 
    return true;
 }
+
+void Machine_scanGPUUsage(Machine* super) {
+   DarwinMachine* dhost = (DarwinMachine *)super;
+   CFMutableDictionaryRef properties = NULL;
+   CFDictionaryRef perfStats;
+   CFNumberRef deviceUtil;
+   int device = 0, ret;
+
+   if (!dhost->GPUService) {
+      return;
+   }
+
+   ret = IORegistryEntryCreateCFProperties(dhost->GPUService, &properties, kCFAllocatorDefault, kNilOptions);
```

Comment:
Declare variables just before their first use. We are targeting for C99 and so there's no need to put declarations before any statement.

```suggestion
   CFMutableDictionaryRef properties = NULL;
   int ret = IORegistryEntryCreateCFProperties(dhost->GPUService, &properties, kCFAllocatorDefault, kNilOptions);
```

By the way, if `ken_return_t` is available from the included headers, then prefer `kern_return_t` to `int`.